bin/rucio

```diff
@@ -1003,13 +1018,28 @@ def upload(args):
     else:
         logger.debug('Skipping dataset registration')
 
+    def make_valid_did(lfn_dict):
```

Comment:
Do we still need the `make_valid_did` method? Also in the rsemanager? It looks like to me that the file dictionary now always consists of `scope`, `name` and `filename`, thus it is not needed anymore?

Reply:
`filename` isn't a valid key in the JSON schema for a DID.  Hence, if we don't remove it, it fails the JSON validation later...

Reply:
Right, understood. In that case I would suggest we add it to rucio.common.utils, so the code doesn't need to be duplicated. Then the patch is ready from my side 🎉  🕺 🎆